Assignment/Program.cs

```diff
@@ -1,7 +1,49 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter a number : ");
+   string decimalNumber = Console.ReadLine ();

```

Comment:
decimalNumber not a good name. Could be _string s_ or _string str_

Reply:
ok